pymeasure/instruments/hp/hp856Xx.py

```diff
@@ -36,12 +36,15 @@
 log.addHandler(logging.NullHandler())
 
 
-class StrEnum(str, Enum):
-    """Until StrEnum is broadly available / pymeasure relies on python <=
-    3.10.x."""
+try:
+    from enum import StrEnum
+except ImportError:
+    class StrEnum(str, Enum):
+        """Until StrEnum is broadly available / pymeasure relies on python <=
+        3.10.x."""
 
-    def __str__(self):
-        return self.value
+        def __str__(self):
+            return self.value
```

Comment:
Is it necessary?

I did just `pass` in my implementation, but that might be different. Afaik it is sufficient to inherit from str and Enum.

If you need it, it is fine.

Reply:
Actually it is kind of "necessary" because the enums string representation would resolve in 'class-name.value' and not just 'value'. I could change all occurances where it is used to accomodate this. What do you think ?

Reply:
> Actually it is kind of "necessary" because the enums string representation would resolve in 'class-name.value' and not just 'value'.

I did not know this, therefore I asked.

Leave it as it is.

Reply:
Actually it is kind of "necessary" because the enums string representation would resolve in 'class-name.value' and not just 'value'. I could change all occurances where it is used to accomodate this. What do you think ?